src/groups/special_orthogonal.jl

```diff
@@ -8,9 +8,14 @@ Special orthogonal group $\mathrm{SO}(n)$ represented by rotation matrices.
 """
 const SpecialOrthogonal{n} = GroupManifold{Rotations{n},MultiplicationOperation}
 
-has_invariant_metric(::SpecialOrthogonal, ::ActionDirection) = Val(true)
+decorator_transparent_dispatch(::typeof(retract!), M::SpecialOrthogonal, q, p, X) = Val(true)
```

Comment:
Are these new forwards, or are these needed because of some behavior change?

Reply:
This is not needed anymore I think, you'Re right. Anyways, the `true` value is an old way anyways.